packages/story-editor/src/components/canvas/rightClickMenu.js

```diff
@@ -58,23 +60,25 @@ const RightClickMenu = () => {
   }, [ref]);
 
   return createPortal(
-    <DirectionAware>
+    <StyleSheetManager stylisPlugins={[]}>
```

Comment:
This is needed because of the portal, right?

Reply:
yea if you check it out rtl on staging right now, and try and click the layers panel and stuff, you can see the positioning bug this is fixing. 

Basically we're explicitly setting the coordinate based on mouse click, but if the positioning component is direction aware, it changes `right` -> `left`. So this removes that weird behavior.